x-pack/legacy/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/create_analytics_form/job_type.tsx

```diff
@@ -33,9 +33,18 @@ export const JobType: FC<Props> = ({ type, setFormState }) => {
     }
   );
 
+  const classificationHelpText = i18n.translate(
+    'xpack.ml.dataframe.analytics.create.classificationHelpText',
+    {
+      defaultMessage:
+        'Classification supports fields that are numeric, boolean, text, keyword and ip. It is also tolerant of missing values. Please use the advanced editor to apply custom options such as num top classes.',
```

Comment:
Suggest editing this to 

```Classification jobs require a source index that is mapped as a table-like data structure and supports fields that are numeric, boolean, text, keyword or ip. Please use the advanced editor to apply custom options such as the prediction field name.```

The help text for Outlier type jobs also needs an edit to something like:

```
Outlier detection jobs require a source index that is mapped as a table-like data structure and will only analyze numeric and boolean fields. Please use the advanced editor to add custom options to the configuration.
```

And for regression:
```
Regression jobs will only analyze numeric fields. Please use the advanced editor to apply custom options such as the prediction field name.
```